src/components/RangeSlider/components/DualThumb/DualThumb.tsx

```diff
@@ -27,26 +30,53 @@ export interface Props extends RangeSliderProps {
   step: number;
 }
 
+interface KeyHandlers {
+  [key: string]: () => void;
+}
+
 const THUMB_SIZE = 24;
 const OUTPUT_TIP_SIZE = 8;
 
 export default class DualThumb extends React.Component<Props, State> {
+  static getDerivedStateFromProps(props: Props, state: State) {
+    const {min, step, max, value} = props;
+    const {prevValue} = state;
+
+    if (isEqual(prevValue, value) === false) {
```

Comment:
I think we prefer early returns.

```suggestion
    if (isEqual(prevValue, value)) {
      return
    }
    
    return {
        prevValue: value,
        valueLower: sanitizeValueLower(value[0], min, step, value[1]),
        valueUpper: sanitizeValueUpper(value[1], max, step, value[0]),
      };
```